lib/proto/google/api/http_pb.js

```diff
@@ -153,15 +153,15 @@ proto.google.api.Http.serializeBinaryToWriter = function(message, writer) {
 
 /**
  * repeated HttpRule rules = 1;
- * @return {!Array.<!proto.google.api.HttpRule>}
+ * @return {!Array<!proto.google.api.HttpRule>}
```

Comment:
I'm curious as to why we're getting diffs like this? @sangaman, is this related to the grpc-tools version changes?

Reply:
Yes I think so, @ImmanuelSegol could you run `npm install` and then `npm run proto` again?